src/parser.js

```diff
@@ -314,25 +315,29 @@ function convert(context) {
 
     switch (type(node)) {
       case 'Value': {
-        let value = convertChild(node.base);
-        node.properties.forEach(prop => {
-          value = accessOpForProperty(value, prop, node.base.locationData);
-          if (value.type === 'MemberAccessOp' && value.expression.type === 'MemberAccessOp') {
-            if (value.expression.memberName === 'prototype' && value.expression.raw.slice(-2) === '::') {
-              // Un-expand shorthand prototype access.
-              value = {
-                type: 'ProtoMemberAccessOp',
-                line: value.line,
-                column: value.column,
-                range: value.range,
-                raw: value.raw,
-                expression: value.expression.expression,
-                memberName: value.memberName
-              };
+        try {
+          return mapValue(context, node);
+        } catch (err) {
```

Comment:
Might be good to re-raise on unexpected runtime errors. Maybe you could put "Not implemented" in every error message where it's known that you want to fall back, and check for that. I think that also would make it easier to spot old try/catch blocks that need to be cleaned up later, in case you forget any.